src/plugins/intel_cpu/tests/functional/single_layer_tests/classes/eltwise.cpp

```diff
@@ -0,0 +1,506 @@
+                ngraph::helpers::EltwiseTypes::ADD,
+                ngraph::helpers::EltwiseTypes::MULTIPLY,
+        #if defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64) // TODO: Fix CVS-105430
+                ngraph::helpers::EltwiseTypes::SUBTRACT,
+        #endif
+                ngraph::helpers::EltwiseTypes::SQUARED_DIFF,
+        };
+        return eltwiseOpTypesBinDyn;
+}
+
+const std::vector<CPUSpecificParams>& cpuParams_4D() {
+        static const std::vector<CPUSpecificParams> cpuParams_4D = {
+                //CPUSpecificParams({nChw16c, nChw16c}, {nChw16c}, {}, {}),
+                CPUSpecificParams({nhwc, nhwc}, {nhwc}, {}, {}),
+                CPUSpecificParams({nchw, nchw}, {nchw}, {}, {})
+        };
+        return cpuParams_4D;
+}
+
+const std::vector<CPUSpecificParams>& cpuParams_5D() {
+        static const std::vector<CPUSpecificParams> cpuParams_5D = {
+                //CPUSpecificParams({nCdhw16c, nCdhw16c}, {nCdhw16c}, {}, {}),
+                CPUSpecificParams({ndhwc, ndhwc}, {ndhwc}, {}, {}),
+                CPUSpecificParams({ncdhw, ncdhw}, {ncdhw}, {}, {})
+        };
+        return cpuParams_5D;
+}
+
+const std::vector<std::vector<ov::Shape>>& inShapes_4D() {
+        static const std::vector<std::vector<ov::Shape>> inShapes_4D = {
+                {{2, 4, 4, 1}},
+                {{2, 17, 5, 4}},
+                {{2, 17, 5, 4}, {1, 17, 1, 1}},
+                {{2, 17, 5, 1}, {1, 17, 1, 4}},
+        };
+        return inShapes_4D;
+}
+
+const std::vector<std::vector<ov::Shape>>& inShapes_5D() {
+        static const std::vector<std::vector<ov::Shape>> inShapes_5D = {
+                {{2, 4, 3, 4, 1}},
+                {{2, 17, 7, 5, 4}},
+                {{2, 17, 6, 5, 4}, {1, 17, 6, 1, 1}},
+                {{2, 17, 6, 5, 1}, {1, 17, 1, 1, 4}},
+        };
+        return inShapes_5D;
+}
+
+const std::vector<ngraph::helpers::EltwiseTypes>& eltwiseOpTypesI32() {
+        static const std::vector<ngraph::helpers::EltwiseTypes> eltwiseOpTypesI32 = {
+                ngraph::helpers::EltwiseTypes::ADD,
+                ngraph::helpers::EltwiseTypes::MULTIPLY,
+        #if defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64) // TODO: Fix CVS-105430
+                ngraph::helpers::EltwiseTypes::SUBTRACT,
+                ngraph::helpers::EltwiseTypes::DIVIDE,
+        #endif
+                ngraph::helpers::EltwiseTypes::SQUARED_DIFF,
+        };
+        return eltwiseOpTypesI32;
+}
+
+
+
+
+const std::vector<ngraph::helpers::InputLayerType>& secondaryInputTypes() {
+        static const std::vector<ngraph::helpers::InputLayerType> secondaryInputTypes = {
+                ngraph::helpers::InputLayerType::CONSTANT,
+                ngraph::helpers::InputLayerType::PARAMETER,
+        };
+        return secondaryInputTypes;
+}
+
+const std::vector<std::vector<ov::Shape>>& inShapes_4D_Planar_Blocked() {
```

Comment:
Blocked only supported and used by x86 scope.
So this definitions should be moved into x86 instance.
The idea is that the main classes/*.cpp contains only common definitions used by all the instances (all the archs)

Reply:
Moved to x64 file